sirepo/template/controls.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+u"""Controls execution template.
+
+:copyright: Copyright (c) 2020 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+from sirepo.template.madx import *
```

Comment:
@robnagler this is the cheapest of inheritance. Do you have thoughts on what might be better?

My ideas:
- There aren't that many methods that need to be implemented. I could just implement them and call the madx equivalents in the method body.
- Move to a class base system and inherit from the madx class.

Reply:
I have a better idea for this, rather than bringing in the mad-x schema and deriving the templates. Instead, just the simple editors need to be in the schema, and the whole mad-x beamline is treated as a json blob within the sirepo-data.json. I'll make some changes to this branch to demonstrate.